components/style/properties/longhand/text.mako.rs

```diff
@@ -75,7 +76,9 @@
             match *self {
                 Side::Clip => dest.write_str("clip"),
                 Side::Ellipsis => dest.write_str("ellipsis"),
-                Side::String(ref s) => dest.write_str(s)
+                Side::String(ref s) => {
+                    cssparser::serialize_string(&**s, dest)
```

Comment:
I think you can also use `CssStringWriter` instead of `cssparser::serialize_string`, but that's ok too.